ios/UploadAttachments/UploadAttachments/StoreManager.m

```diff
@@ -150,6 +150,19 @@ -(NSString *)getToken {
   return [credentials objectForKey:@"token"];
 }
 
+-(UInt64)getMaxImageSize {
```

Comment:
this is basically a copy of `getMaxFileSize` perhaps have this as a base function?